src/components/app/Home.js

```diff
@@ -44,17 +44,16 @@ const InstallButton = ({
 type InstallButtonProps = {
   name: string,
   xpiUrl: string,
-  children?: React.Element<*>,
+  children?: React.Node,
   className?: string,
 };
 
 type UploadButtonProps = {
   retrieveProfileFromFile: File => void,
 };
 
-class UploadButton extends PureComponent {
-  props: UploadButtonProps;
-  _input: HTMLInputElement;
+class UploadButton extends React.PureComponent<UploadButtonProps> {
+  _input: HTMLInputElement | null;
```

Comment:
Flow now forces us to properly type references: they actually can be null !

Reply:
Yay, strictness!